src/services/address/addressLookUp.ts

```diff
@@ -69,6 +70,41 @@ export class AddressLookUpService {
         });
     }
 
+    public getAddressFromPostcodeUpdateJourney (req: Request, postcode: string, inputPremise: string, acspDetails: AcspFullProfile, businessAddress: boolean, ...nexPageUrls: string[]) : Promise<string> {
```

Comment:
You are saving the address so I think this method should not be called as get...may be its more like processAddressFromPostcodeUpdateJourney